megamek/src/megamek/client/bot/princess/FireControl.java

```diff
@@ -3220,6 +3221,18 @@ Mounted getAntiAirAmmo(final List<Mounted> ammoList,
         return unjamVector;
     }
 
+    /**
+     * Return a "Find Club" action, if the unit in question can find a club.
+     */
+    public FindClubAction getFindClubAction(Entity shooter) {
+        if (FindClubAction.canMechFindClub(shooter.getGame(), shooter.getId())) {
+            FindClubAction findClubAction = new FindClubAction(shooter.getId());
+            return findClubAction;
+        }
+        
+        return null;
```

Comment:
```suggestion
        return FindClubAction.canMechFindClub(shooter.getGame(), shooter.getId()) ? new FindClubAction(shooter.getId()) : null;
```

Reply:
I don't really find ternary operators more readable in this particular situation.